src/CheckoutSdk/Reconcilation/ReconciliationClient.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Checkout.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Reconciliation
+{
+    public class ReconciliationClient : AbstractClient, IReconciliationClient
+    {
+        private const string ReportingPath = "reporting";
+        private const string PaymentsPath = "payments";
+        private const string StatementsPath = "statements";
+        private const string DownloadPath = "download";
+
+        public ReconciliationClient(IApiClient apiClient, CheckoutConfiguration configuration) : base(apiClient, configuration, SdkAuthorizationType.SecretKey)
+        {
+        }
+
+        public Task<ReconciliationPaymentReportResponse> QueryPaymentsReport(ReconciliationQueryPaymentsFilter filter)
```

Comment:
There is a `CheckoutUtils.ValidateParams` method. Please add for all the incoming params like `id's` or `request`.
Also I believe that all the clients have the support for `CancellationToken`